R/module_nested_tabs.R

```diff
@@ -336,33 +336,26 @@ srv_teal_module.teal_module <- function(id,
         data = data,
         is_active = is_active
       )
-      is_transform_failed <- reactiveValues()
       transformed_teal_data <- srv_transform_teal_data(
         "data_transform",
         data = filtered_teal_data,
         transformators = modules$transformators,
-        modules = modules,
-        is_transform_failed = is_transform_failed
+        modules = modules
       )
       any_transform_failed <- reactive({
-        any(unlist(reactiveValuesToList(is_transform_failed)))
+        !inherits(try(transformed_teal_data(), silent = TRUE), "teal_data") &&
+          inherits(filtered_teal_data(), "teal_data")
```

Comment:
As `transformed_teal_data` depends on the output of `filtered_teal_data` it can be omitted and I think we would get the same result (at least on the app on the issue it works as before). 

```suggestion
        !inherits(try(transformed_teal_data(), silent = TRUE), "teal_data")
```